src/components/WarningModal/WarningModal.js

```diff
@@ -25,15 +25,18 @@ class WarningModal extends React.PureComponent {
     message: PropTypes.string,
     onConfirm: PropTypes.func,
     onCancel: PropTypes.func,
+    keepOpen: PropTypes.array,
   };
 
   componentDidUpdate(prevProps) {
-    if (!prevProps.isOpen && this.props.isOpen) {
+    const { keepOpen, isOpen, closeElements } = this.props;
+
+    if (!prevProps.isOpen && isOpen) {
       // TODO want to do make something like'closeOtherElements', but that for latter
-      const popUpToClose = getPopupElements().filter(
-        ele => ele !== 'warningModal',
+      const popupElementsToClose = getPopupElements().filter(
+        ele => ele !== 'warningModal' && (!keepOpen || !keepOpen.includes(ele)),
```

Comment:
Can we tell me why we wanted to close the left panel when the warning modal is showing up? I feel like we can just keep the panels open... If we do that we don't need this extra logic.

Reply:
There was no reason for us to have it,  want to keep the left panel open when the warning (and error, which already doing this ) modal come up.

I update the `componentDidUpdate` function to filter out left panel and removed the "keepOpen" logic. I was thinking that logic would be good for later but we'll see if "later" ever happens